__tests__/index.test.js

```diff
@@ -197,8 +199,88 @@ describe("/api/articles/:article_id/comments tests", () => {
       .get("/api/articles/nine/comments")
       .expect(400)
       .then((response) => {
-        console.log(response.body.message);
         expect(response.body.message).toBe("Bad request.");
       });
   });
 });
+
+describe("POST:/ api/articles/:article_id/comments tests", () => {
```

Comment:
Almost perfect testing here - just a couple of points but you have covered everything off well 👍 
A further test could be that the endpoint responds with a Status 201, ignores unnecessary properties so long as the required ones are included on the request.